### PR TITLE
Fixes to the getting started commands

### PR DESCRIPTION
**Pull request by kikisaints**
Fixes RNW#[4132](https://github.com/microsoft/react-native-windows/issues/4132) and addresses [comment here](https://github.com/microsoft/react-native-windows-samples/pull/161#issuecomment-656776055)

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ For information around how to set up React Native, see the [React Native Getting
 Remember to call `react-native init` from the place you want your project directory to live.
 
 ```
-npx react-native init <projectName> --version ^0.62.2
+npx react-native init <projectName> --version 0.62.2
 ```
->To create TypeScript template, run `npx react-native init <projectName> --version ^0.62.2 --template react-native-template-typescript`.<br><br>
+>To create TypeScript template, run `npx react-native init <projectName> --version 0.62.2 --template react-native-template-typescript`.<br><br>
```
Comment (acoates-ms) [inline docs/getting-started.md:19]: This should be `npx --ignore-existing react-native init <projectName> --template react-native-template-typescript@6.4.*`

File: docs/getting-started.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ For information around how to set up React Native, see the [React Native Getting
...
 
 ```
-npx react-native init <projectName> --version ^0.62.2
+npx react-native init <projectName> --version 0.62.2
 ```
->To create TypeScript template, run `npx react-native init <projectName> --version ^0.62.2 --template react-native-template-typescript`.<br><br>
+>To create TypeScript template, run `npx react-native init <projectName> --version 0.62.2 --template react-native-template-typescript`.<br><br>
 > If you've installed react native globally in the past, via `npm install -g react-native`, and are having issues with the new instructions, try running:<br>
-> `npx --ignore-existing react-native init <projectName> --template react-native@^0.62.2` instead.
+> `npx --ignore-existing react-native init <projectName> --template react-native-template-typescript@6.4.*` instead.
```
Comment (acoates-ms) [inline docs/getting-started.md:21]: This should be `npx --ignore-existing react-native init <projectName> --template react-native@^0.62.2`

File: docs/getting-started.md [hunk 3]
```diff
@@ -14,11 +14,11 @@ For information around how to set up React Native, see the [React Native Getting
 Remember to call `react-native init` from the place you want your project directory to live.
 
 ```
-npx react-native init <projectName> --version ^0.62.2
+npx react-native init <projectName> --version 0.62.2
```
Comment (acoates-ms) [inline docs/getting-started.md:17]: This should be `npx react-native init <projectName> --template react-native@^0.62.2`

File: docs/getting-started.md (hunk 1 above)
Comment (kikisaints) [reply docs/getting-started.md]: Should be updated now!

File: docs/getting-started.md (hunk 3 above)
Comment (kikisaints) [reply docs/getting-started.md]: fixed!

File: docs/getting-started.md (hunk 2 above)
Comment (kikisaints) [reply docs/getting-started.md]: updated and fixed.

**Pull request merged**